test/e2e/util/fixture.go

```diff
@@ -247,15 +247,17 @@ func teardown(ctx context.Context, t *testing.T, client crclient.Client, hc *hyp
 				HypershiftOperatorInfoName,
 				nodepoolmetrics.NodePoolSizeMetricName,
 				nodepoolmetrics.NodePoolAvailableReplicasMetricName,
+				nodepoolmetrics.NodePoolDeletionDurationMetricName,
+				nodepoolmetrics.NodePoolInitialRolloutDurationMetricName,
 			} {
 				// Query fo HC specific metrics by hc.name.
 				query := fmt.Sprintf("%v{name=\"%s\"}", metricName, hc.Name)
 				if metricName == HypershiftOperatorInfoName {
 					// Query HO info metric
 					query = HypershiftOperatorInfoName
 				}
-				if metricName == nodepoolmetrics.NodePoolSizeMetricName || metricName == nodepoolmetrics.NodePoolAvailableReplicasMetricName {
-					// Mulham: `namespace`` label is used and overrwriten by prometheus relabeling configs.
+				if strings.HasPrefix(metricName, "hypershift_nodepools") {
```

Comment:
why is this different for the original query in line 252?

Reply:
the `name` label for nodepools metrics is the nodepool name and not the HC, so the original metric wouldn't find nodepools metrics by name.